.github/workflows/freshen-images/lib.sh

```diff
@@ -82,16 +89,24 @@ function build_generic() {
   local tag=$1
   local id=$2
   local platforms=$3
+  local buildlatest=$4
+  #doesnt pushing to this tag update latest?
```

Comment:
I think this pushes to `vX.Y` instead of `vX.Y.Z`. For example, when rebuilding tag `v1.25.2`, `tag_maj_min` would end up being `v1.25`

Reply:
woops, left that comment in from when i was working on it.